src/blocking.rs

```diff
@@ -228,7 +231,14 @@ impl BlockingIronOxide {
         user_create_opts: &UserCreateOpts,
     ) -> Result<UserCreateResult> {
         let rt = create_runtime();
-        rt.enter(|| block_on(IronOxide::user_create(jwt, password, user_create_opts)))
+        rt.enter(|| {
+            block_on(IronOxide::user_create(
+                jwt,
+                password,
+                user_create_opts,
+                None,
```

Comment:
Shouldn't this function take a `timeout` like `generate_new_device()` and `user_verify()`?

Reply:
Yes. Great catch.